src/scanoss/cli.py

```
@@ -623,24 +624,16 @@ def setup_args() -> None:  # noqa: PLR0912, PLR0915
-        help='Enable best match mode (optional - default: False)',
-    )
-    p_folder_scan.add_argument(
-        '--threshold',
+        '--rank-threshold',
         type=int,
-        choices=range(1, 101),
-        metavar='1-100',
-        default=100,
-        help='Threshold for result matching (optional - default: 100)',
+        default=DEFAULT_HFH_RANK_THRESHOLD,
+        help='Filter results to only show those with rank value at or below this threshold (e.g., --rank-threshold 3 returns results with rank 1, 2, or 3). '
```

Ruff (0.11.9)

Line too long (157 > 120)

(E501)